### PR TITLE
8263760: Update gradle to version 7.0.1

### PR DESCRIPTION
**Pull request by arapte**
There were two minor merge conflicts in `build.gradle` and one conflict in `gradle/wrapper/gradle-wrapper.properties` while cherry picking the change. 
And it also required similar changes to be made for project fxpackager. The [first commit](https://github.com/openjdk/jfx11u/pull/49/commits/7700a6d1a7c82df8fc269f1de6f3bd5119fb3397) is the original cherry pick and [second commit](https://github.com/openjdk/jfx11u/pull/49/commits/cea09bdfc96374ce00560c13e6730ab76f8b20e9) is the additional changes for fxpackager.
Verified the build on all three platforms with gradle 7.0.1 and apache ant 1.10.5

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issues
 * [JDK-8263760](https://bugs.openjdk.java.net/browse/JDK-8263760): Update gradle to version 7.0.1
 * [JDK-8240336](https://bugs.openjdk.java.net/browse/JDK-8240336): JavaFX build uses deprecated features that will be removed in gradle 7
 * [JDK-8262236](https://bugs.openjdk.java.net/browse/JDK-8262236): Configure Gradle checksum verification

### Reviewers
 * [Kevin Rushforth](https://openjdk.java.net/census#kcr) (@kevinrushforth - **Reviewer**)
 * [Johan Vos](https://openjdk.java.net/census#jvos) (@johanvos - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jfx11u pull/49/head:pull/49` \
`$ git checkout pull/49`

Update a local copy of the PR: \
`$ git checkout pull/49` \
`$ git pull https://git.openjdk.java.net/jfx11u pull/49/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 49`

View PR using the GUI difftool: \
`$ git pr show -t 49`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jfx11u/pull/49.diff">https://git.openjdk.java.net/jfx11u/pull/49.diff</a>

</details>

### CODE REVIEW

**Review by kevinrushforth — COMMENTED** [review]
Looks fine with a couple comments. I tested it with the older gradle `6.3` as well as `7.0.1`.

File: build.gradle [hunk 1]
```diff
@@ -2755,9 +2755,9 @@ project(":fxpackager") {
     }
 
     dependencies {
-        antpluginCompile group: "org.apache.ant", name: "ant", version: "1.8.2"
+        implementation group: "org.apache.ant", name: "ant", version: "1.8.2"
```
[the remark this answers is not in the document]
Comment (arapte) [reply build.gradle]: This is corrected in next commit.

**Review by kevinrushforth — APPROVED** [review]

**Comment by kevinrushforth** [comment]
@johanvos can you also review this?

**Comment by johanvos** [comment]
Already did, but forgot to look at the results. Looks good to me. We might remove the fxpackager part, but it doesn't hurt being there either.

**Review by johanvos — APPROVED** [review]

**Comment by kevinrushforth** [comment]
Thanks.

> We might remove the fxpackager part, but it doesn't hurt being there either.

Since they aren't being built (and might no longer be buildable), we could consider backporting [JDK-8203379](https://bugs.openjdk.java.net/browse/JDK-8203379) to jfx11u remove the sources in a future update release. As it is, these sources serve no purpose other than as a source of "technical debt".

**Pull request closed without merging**